### DISCUSSION TITLE
The output contains unreasonable parts

### DISCUSSION THREAD

**Discussion by CoazMoxuan** [openai/whisper#2475] [Q&A]
There is a question included at the end of my audio file, and the output will contain a duplicate answer to the question.
[1.zip](https://github.com/user-attachments/files/18230048/1.zip)

**Comment by misutoneko** [comment]
This kind of material seems ideally suited for VAD preprocessing.

Other than that, these parameters seem to help with Turbo:
--word_timestamps True --compression_ratio_threshold 1.2

but this solution is a bit unsatisfactory because you wouldn't know the value beforehand.
